### PR TITLE
[Enhancement] Enable chunk_size session variable for pipeline engine

### PR DESCRIPTION
**Pull request by liuyehcf**
## What I'm doing:

* Enable `chunk_size` session variable for pipeline engine.
* Echo with #40756. Check validality when setting chunk_size.

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by stdpain — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java [hunk 1]
```diff
@@ -865,8 +865,6 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = CHUNK_SIZE, flag = VariableMgr.INVISIBLE)
     private int chunkSize = 4096;
```
Comment (fzhedu) [inline fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:866]: what abot `batch_size`, clean it?

Comment (silverbullet233) [reply fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:866]: how about adding a check in FE? just return error if chunk_size is not in range of (0,65535]

Comment (liuyehcf) [reply fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:866]: Ok, seems better.

Comment (liuyehcf) [reply fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:866]: > what abot `batch_size`, clean it?

batch_size is marked as deprecated.

**Review by stdpain — APPROVED** [review]

**Review by silverbullet233 — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Comment by liuyehcf** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**